### PR TITLE
Foreman fog proxmox 0.16.1 rpm develop

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

**Comment by evgeni** [comment]
[test rpm-copr]

File: packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec [hunk 1]
```diff
@@ -88,30 +73,35 @@ cp -pa .%{gem_dir}/* \
...
 %{gem_instdir}/test
 
-%post
-/sbin/semanage port -a -t http_port_t -p tcp 8006 &> /dev/null || :
-
-%postun
-if [ $1 -eq 0 ] ; then
-  /sbin/semanage port -d -t http_port_t -p tcp 8006 &> /dev/null || :
-fi
-
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec:110]: I think this should be kept

File: packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec [hunk 2]
```diff
@@ -27,54 +23,43 @@ BuildRequires: (rubygem(fog-proxmox) >= 0.15 with rubygem(fog-proxmox) < 1)
 BuildArch: noarch
 Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
-%if 0%{?fedora} >= 27 || 0%{?rhel} >= 8
-Requires(post):   policycoreutils-python-utils
-Requires(postun): policycoreutils-python-utils
-%else
-Requires(post):   policycoreutils-python
-Requires(postun): policycoreutils-python
-%endif
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec:36]: I think this should be kept
(but probably cleaned up, the else branch is useless now)

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec (hunk 1 above)
Comment (nadjaheitmann) [reply packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec]: True, changed it, thanks for paying attention!

File: packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec [hunk 3]
```diff
@@ -112,6 +104,15 @@ fi
...
 
 %changelog
+* Wed Jul 24 2024 Nadja Heitmann <nadjah@atix.de> 0.16.1-1
+- Update to 0.16.1-1
+
+* Wed Jul 24 2024 Nadja Heitmann <nadjah@atix.de> - 0.16.1-1
+- Update to 0.16.1
+
+* Wed Jul 24 2024 Nadja Heitmann <nadjah@atix.de> - 0.16.0-1
+- Update to 0.16.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec:114]: I think you can fold the changelog entries into one (and squash the commits), and then we're good to go here.

**Review by evgeni — APPROVED** [review]

**Pull request merged**